ansible/configs/ocs4-external-implementation/post_infra.yml

```diff
@@ -21,3 +21,11 @@
     - name: OpenShift Floating IPs on OpenStack 
       include_role:
         name: ocp-infra-osp-fip
+    
+    - name: find ocp network subnet id
+      shell: openstack subnet list -f csv |grep {{ guid }}-ocp-subnet |cut -f1 -d,
```

Comment:
Rather than use `-f csv` with `cut`, perhaps use `-f json` and use filter `from_json`?

Reply:
Thank you @jkupferer, I will check how I can use that instead cut.